src/AIS_Decoder.cpp

```diff
@@ -3215,11 +3193,11 @@ void AIS_Decoder::OnTimerAIS(wxTimerEvent &event) {
 }
 
 AIS_Target_Data *AIS_Decoder::Get_Target_Data_From_MMSI(int mmsi) {
-  if (AISTargetList->find(mmsi) ==
-      AISTargetList->end())  // if entry does not exist....
+  if (AISTargetList.find(mmsi) ==
+      AISTargetList.end())  // if entry does not exist....
```

Comment:
While you are on it: Putting the comment on a separate line makes cmake-format produce something more sane.

Reply:
Do you mind if I just drop these? They don't seem very informative.

Reply:
Went ahead and did this, and applied clang-format to all of my changes.